src/main/java/domain/Prize.java

```diff
@@ -0,0 +1,36 @@
+package domain;
+
+public enum Prize {
+    THIRD(3, false, 5000),
+    FOURTH(4, false, 50000),
+    FIFTH(5, false, 1500000),
+    SECOND(5, true, 30000000),
+    FIRST(6, false, 2000000000L);
```

Comment:
이후 루루 코드를 참고하면서 코멘트 반영할 때 발견한건데 실제로는 이렇게 네이밍이 되어야 할 것 같다는 생각도 들었어요 -- `FIRST` 와 같은 이름이 순위를 의미하는 것이라면요
```suggestion
    FIFTH(3, false, 5000),
    FOURTH(4, false, 50000),
    THIRD(5, false, 1500000),
    SECOND(5, true, 30000000),
    FIRST(6, false, 2000000000L);
```